docs/DesignDocs/SGX_Endorsements_V2.md

```diff
@@ -0,0 +1,254 @@
+
+### Get QE Identity Info
+1. The 'qeIdentity' block was renamed to 'enclaveIdentity'.
+2. A new field, enclaveIdentity:id was added to differentiate QE Id info from Quoting Enclave (QE) and Quoting Validation Enclave (QVE).
+3. enclaveIdentity:version changed from 1 to 2.
+4. New field enclaveIdentity:tcbEvaluationDataNumber.
+5. qeIdentity:isvsvn moved to enclaveIdentity:tcbLevels:tcb:isvsvn.
+6. New fields in enclaveIdentity:tcbLevels named tcbData, tcbStatus and advisoryIDs.
+
+### New field definitions
+
+#### tcbType:
+	type: integer
+	example: 0
+	description: >- Type of TCB level composition that determines TCB level comparison logic
+
+##### tcbEvaluationDataNumber:
+	type: integer
+	example: 2
+	description: >- A monotonically increasing sequence number changed when Intel updates the content of the TCB evaluation data set: TCB Info, QE Idenity and QVE Identity. The tcbEvaluationDataNumber update is synchronized across TCB Info for all flavors of SGX CPUs (Family-Model-Stepping-Platform-CustomSKU) and QE/QVE Identity. This sequence number allows users to easily determine when a particular TCB Info/QE Idenity/QVE Identiy superseedes another TCB Info/QE Identity/QVE Identity (value: current TCB Recovery event number stored in the database).
+
+##### tcbDate:
+	type: string
+	format: date-time
+	description: >- Representation of date and time when the TCB level was certified not to be vulnerable to any issues described in Security Advisories (SAs) that were published on or prior to this date. The time shall be in UTC and the encoding shall be compliant to ISO 8601 standard (YYYY-MM-DDThh:mm:ssZ).
+
+##### tcbStatus:
+	type: string
+	description: TCB level status
+
+##### advisoryIDs:
+	type: array
+	description: >- Array of Advisory IDs describing vulnerabilities that this TCB level is vulnerable to. Note: The value can be different for different FMSPCs. This field is optional. It will be present only if the list of Advisory IDs is not empty.
+	items:
+        type: string
+
+For more information on the new fields please see [API Version 2](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate-v2).
+
+
+## Changes
+
+### Update current structs with new additional fields and create new type definitions.
+
+```C
+#define OE_TCB_LEVEL_STATUS_UNKNOWN                 (0)
+#define OE_TCB_LEVEL_STATUS_REVOKED                 (1 << 0)
+#define OE_TCB_LEVEL_STATUS_OUT_OF_DATE             (1 << 1)
+#define OE_TCB_LEVEL_STATUS_CONFIGURATION_NEEDED    (1 << 2)
+#define OE_TCB_LEVEL_STATUS_UP_TO_DATE              (1 << 3)
+#define OE_TCB_LEVEL_STATUS_QE_IDENTITY_OUT_OF_DATE (1 << 4)
+
+/*! \struct oe_tcb_level_status_t
+ */
+typedef union _oe_tcb_level_status
+{
+    struct {
+        uint32_t revoked : 1;                       //! "Revoked"
+        uint32_t outofdate : 1;                     //! "OutOfDate"
+        uint32_t configuration_needed :1;           //! "ConfigurationNeeded"
```

Comment:
nit: would be nice to use consistent spacing, i.e. space after : like in the rest of the lines

Reply:
Good catch. Fixed typo.